recipes-bsp/u-boot/files/0027-arm-dts-iot2050-Add-support-for-M.2-variant.patch

```diff
@@ -114,14 +122,34 @@ index 0000000000..741b9177da
 +	>;
 +};
 +
++/*
++ * Default configuration for B-key with 2 PCIe lanes, E-key disabled.
```

Comment:
Default is B-key with usb3.0 and E-key with pcie one lane.
even if B-key with two lane, E-key could also use 2.0

Reply:
Nope, this was changed in order to make overlays possible: U-Boot does not support removing properties via overlays. Therefore, the baseline DT must start in a way that we only add or change properties.

Reply:
Oh, that the E-key can have a role when the B-key uses two PCI lanes, that was nowhere mentioned so far. Can you elaborate please?

Reply:
OK, will update the comment and outputs accordingly.

Reply:
The title in iot2050setup should be refined to `M.2 PCIE/USB3.0 Configuration`